examples/step-20/step-20.cc

```diff
@@ -17,36 +17,39 @@
  * Author: Wolfgang Bangerth, Texas A&M University, 2005, 2006
  */
 
-
 // @sect3{Include files}
 
 // Since this program is only an adaptation of step-4, there is not much new
 // stuff in terms of header files. In deal.II, we usually list include files
 // in the order base-lac-grid-dofs-fe-numerics, followed by C++ standard
 // include files:
```

Comment:
we need to change this comment :)

Reply:
But there was a reason for this list. It goes through these directories in roughly a breadth-first traversal of a dependency tree.

Reply:
If we want to keep this we should (as @tamiko mentioned) just put a different `.clang-format` file in this directory and handle things manually: I would be okay with doing that.